run-build.sh

```diff
@@ -93,11 +93,18 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --env-vars)
```

Comment:
@am11 Yes, I know. But just wanted to modify it to the same structure as core-setup. The core-setup build process has recently changed, so this patch should wait a bit longer.